.github/workflows/codeql-analysis.yml

```diff
@@ -13,10 +13,10 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "1.6" ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ "main" ]
+    branches: [ "1.6" ]
   schedule:
     - cron: '31 14 * * 1'
```

Comment:
I was thinking about this when I was doing the changes in the snap repo and I wonder why don't we run that on every push anywhere? I mean it doesn't take long and can provide valuable feedback.

Reply:
It makes sense.
But in that case we need to make that change to the main branch I guess.

Reply:
Done.